api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -143,11 +133,17 @@ public void setUserService(UserService userService) {
           "User billing project is not yet initialized, cannot list/create clusters");
     }
 
+    String clusterName = LeonardoNotebooksClient.DEFAULT_CLUSTER_NAME;
+    if (workbenchConfigProvider.get().featureFlags.useBillingProjectBuffer) {
+      // With 1 billing project == 1 workspace, a billing project may contain multiple clusters (up
```

Comment:
Not related to these changes but I think we should have a term to refer to the `1 billing project == 1 workspace` design/work just so its easy to refer to in our documentation.

Reply:
Maybe project-per-workspace is the shortest (yet still clear) description, or 1PPW as an acronym... neither is elegant, but as long as we all know what it means.

I tried linking http://broad.io/1ppw to the core design doc (which itself can link to Jira epics, tickets, etc.). Maybe that helps?

Reply:
Good point... this is what I've been calling it, but it's a bit of a mouthful. I think it's probably solidified enough that we shouldn't change it, though in hindsight something like "hermetic workspaces" might have been less choppy, albeit more opaque.

Updated this comment to match verbage.

Reply:
I like 1PPW, I wasn't aware of that acronym actually.

Reply:
Hah - I just meant the "1 project per workspace" part, I think Greg just invented the acronym. Would shorten for typing in comments at least.